target/linux/ath79/dts/ar9132_tplink_tl-wa901nd-v2.dts

```diff
@@ -87,16 +87,26 @@
 				};
 			};
 
-			partition@1 {
+			partition@20000 {
```

Comment:
This should also be moved to a new commit.

Reply:
Done.